config/settings/base.py

```diff
@@ -509,6 +509,10 @@
     "NPTG_IMPORT_URL", default="https://naptan.api.dft.gov.uk/v1/nptg"
 )
 
+BANK_HOLIDAY_API_URL = env(
```

Comment:
Please update .env templates with this entry.

Reply:
As default value is provided, is the .env entry required?

Reply:
For this to be managed on cloud as env variables. This .env templates serves as an example.

Reply:
done